### PR TITLE
report(flow): summary page

### PR DESCRIPTION
**Pull request by adamraine**
Still a bit messy, and I haven't started the tests yet. Wanted to get some initial thoughts on the structure / looks before I finish up.

`Gauge` is an example of a component imported from the normal report renderer.

### CODE REVIEW

File: flow-report/assets/styles.css [hunk 1]
```diff
@@ -9,6 +9,9 @@
   --sidebar-title-font-size: 16px;
   --sidebar-summary-font-size: 14px;
   --sidebar-flow-step-navigation-font-size: 14px;
+  --summary-flow-step-label-font-size: 16px;
```
Comment (connorjclark) [inline flow-report/assets/styles.css:12]: can you sort these variables, like we do in `styles.css`?

File: flow-report/src/summary/summary.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      <div
+        className="SummaryFlowIcon__line"
+        style={hideTopLine ? {background: 'transparent'} : undefined}
+      />
+      {
+        mode && <FlowStepIcon mode={mode}/>
+      }
+      <div
+        className="SummaryFlowIcon__line"
+        style={hideBottomLine ? {background: 'transparent'} : undefined}
```
Comment (connorjclark) [inline flow-report/src/summary/summary.tsx:31]: could this instead be done with CSS :first-child and :last-child ?

Comment (adamraine) [reply flow-report/src/summary/summary.tsx]: Yes, will probably update `SidebarFlow` with this as well.

**Review by patrickhulce — COMMENTED** [review]
flushing my comments for now, I think overall structure is good. we'll likely want to split up summary/common in the future and wrappers folder seems worth doing already, but happy to move to cleanup and test 👍

**Review by patrickhulce — APPROVED** [review]
nice! great covering a lot of ground here

**Review by bassidi767-wq — COMMENTED** [review]
اا

**Pull request merged**